Wikipedia/Code/MWKDataStore.m

```diff
@@ -1561,6 +1561,16 @@ - (void)clearMemoryCache {
     [self.articlePreviewCache removeAllObjects];
 }
 
+- (void)clearImageAndArticlesSavedOnDiskCache {
```

Comment:
Could we name this a bit differently to convey that it only deletes cached article and image data for `unsaved` articles?
Maybe something like `clearCachesForUnsavedArticles`?

Reply:
(updated after chat confimation)